spec/app/utils/sitesSpec.js

```diff
@@ -0,0 +1,35 @@
+// import {onSitesWithinRadius, addSiteLocation, getLocation, mockLocation} from '../../../app/utils/geo';
+import {geoRef, fireRef} from '../../../config';
+import {getSiteByKey, addSite} from '../../../app/utils/sites';
+
+describe('site helpers', () => {
+  let _key;
+  const _siteInfo = {
+    name: 'Cradle',
+    artist: 'dont care',
+    category: 'Architecture',
+    tags: ['shiny', 'corporate', 'touristy'],
+    description: 'a bunch of shiny balls hanging off a wall',
+    coords: {latitude: 34.0147601, longitude: -118.4934095},
+    imageUrl: 'http://i.imgur.com/tDO1tmL.jpg',
+  };
+
+  it('adds a site', done => {
```

Comment:
Let's choose between using `(done)` (from above) or just `done` (here) to be consistent.